XamlControlsGallery/ControlPages/ToolTipPage.xaml

```diff
@@ -34,19 +35,39 @@
             <local:ControlExample.Example>
                 <TextBlock Text="TextBlock with an offset ToolTip.">
                     <ToolTipService.ToolTip>
-                        <ToolTip Content="Offset ToolTip." HorizontalOffset="100" VerticalOffset="-80"/>
+                        <ToolTip Content="Offset ToolTip." VerticalOffset="-80"/>
                     </ToolTipService.ToolTip>
                 </TextBlock>
             </local:ControlExample.Example>
             <local:ControlExample.Xaml>
                 <x:String xml:space="preserve">
 &lt;TextBlock Text="TextBlock with an offset ToolTip."/&gt;
     &lt;ToolTipService.ToolTip/&gt;
-        &lt;ToolTip Content="Offset ToolTip." HorizontalOffset="100" VerticalOffset="-80"/&gt;
+        &lt;ToolTip Content="Offset ToolTip." VerticalOffset="-80"/&gt;
     &lt;/ToolTipService.ToolTip/&gt;
 &lt;/TextBlock&gt;
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+
+        <local:ControlExample HeaderText="An Image with a ToolTip using PlacementRect." MinimumUniversalAPIContract="7">
+            <local:ControlExample.Example>
+                <Image x:Name="textBoxToPlace" Source="/Assets/SampleMedia/cliff.jpg" Width="400" Height="266" >
+                    <ToolTipService.ToolTip >
+                            <ToolTip Content="PlacementRect Tooltip"  Placement="Right"
```

Comment:
```suggestion
                            <ToolTip Content="Non-occluding ToolTip."  Placement="Right"
```

Ideally, the tooltip and the sample code's content should match :)

Reply:
Haha yes ^^. Thanks for pointing that out!